### DISCUSSION TITLE
Agentic Workflow

### DISCUSSION THREAD

**Discussion by adeel2020** [open-webui/open-webui#2546] [Ideas]
**Is your feature request related to a problem? Please describe.**
There is no functionality to work with agents. There is no option to add agents or create workflow using those agents. 

**Describe the solution you'd like**
I need to perform agentic workflow which can allow to add skillset and agents, create workflow and use the prompt engineering to trigger the desired workflow. Like using Autogen or Crew AI integration would enhance not just the experience but fulfill a much needed functionality

**Describe alternatives you've considered**
Need to add the code manually at different layers to be able to achieve the same functionality

**Additional context**
https://autogen-studio.com/

**Comment by tjbck** [comment]
Related https://github.com/open-webui/pipelines
